easycord/_bot_guild.py

```
@@ -0,0 +1,175 @@
+        image_path: str,
+        *,
+        reason: str | None = None,
+    ) -> discord.Emoji:
+        """Create a custom emoji in a guild from a local image file.
+
+        Raises ``RuntimeError`` if the bot is not in the guild.
+        """
+        guild = self.get_guild(guild_id)
+        if guild is None:
+            raise RuntimeError(f"Bot is not in guild {guild_id}")
+        with open(image_path, "rb") as f:
```

Ruff (0.15.10)

[warning] Async functions should not open files with blocking methods like `open`

(ASYNC230)